requirements2.txt

```diff
@@ -0,0 +1,6 @@
+requests <= 2.24.0
```

Comment:
It's also incorrect to say we want requests less than 2.24, because a really outdated unsupported version can satisfy this.

Reply:
we can just tie it to 2.24.   the >= 2.24 <= 2.27 which seems to be python2 supported versions failed in testing.  <= 2.24.0 will chose 2.24.0 if installing.    I use requirements (on baremetal) to install in our own directory and not in system.  then set PYTHONPATH.